src/libs/actions/BankAccounts.js

```diff
@@ -330,7 +330,12 @@ function fetchUserWallet() {
 function fetchFreePlanVerifiedBankAccount(stepToOpen) {
     // We are using set here since we will rely on data from the server (not local data) to populate the VBA flow
     // and determine which step to navigate to.
-    Onyx.set(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {loading: true});
+    Onyx.set(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {
+        loading: true,
+
+        // We temporarily keep the achData state to prevent UI changes while fetching.
```

Comment:
I'm still confused about why we are losing this value and need to explicitly set it here.

Reply:
We are using `Onyx.set` here, wiping everything and setting that key to only be equal to `{loading: true}`.

Reply:
Ah ok got it now. That's the part I was missing. I think this works for now. Probably all of this API logic should be improved eventually.